src/driver/mod.rs

```diff
@@ -37,19 +42,26 @@ mod util;
 
 mod write;
 
+mod write_fixed;
+
 mod writev;
 
 use io_uring::IoUring;
 use slab::Slab;
+use std::cell::RefCell;
 use std::io;
 use std::os::unix::io::{AsRawFd, RawFd};
+use std::rc::Rc;
 
 pub(crate) struct Driver {
     /// In-flight operations
     ops: Ops,
 
     /// IoUring bindings
     pub(crate) uring: IoUring,
+
+    /// Reference to the currently registered buffers
+    buffers: Option<Rc<RefCell<Buffers>>>,
```

Comment:
This is one of the big deals. Adding to the core Driver.

It's not needed for using the registry, as there are no general functions yet for checking out a buffer that don't require the registry itself.

But it probably provides an important safety guaranteeing the registery won't be dropped without the entire driver being dropped.

So would you mind naming it fixed_buffers, or fixed_registry, or something similar? The term 'buffers' is going to be all over the place in this crate.